webapp/src/filtering/filtering.tsx

```diff
@@ -1,7 +1,17 @@
+import { responsePathAsArray } from "graphql";
 import { useEffect, useRef, useState } from "react"
-import replayArray from "./replays"
+import rankImages from './ranks';
+
+const replayArray = await fetch("http://localhost:5001/parse/replayList").then((response) => response.json())
+const arenaArray = await fetch("http://localhost:5001/parse/all/arena").then((response) => response.json())
+const rankArray = await fetch("http://localhost:5001/parse/all/ranking").then((response) => response.json())
+const durationArray = await fetch("http://localhost:5001/parse/all/duration").then((response) => response.json())
```

Comment:
I understand that right now, we don't really have much data in the database, but this is not a great practice. Better to use pagination, so I am writing this to remind us to paginate (which the API will eventually support)